package/pigpio/pigpio.mk

```diff
@@ -0,0 +1,20 @@
+################################################################################
+#
+# pigpio
+#
+################################################################################
+
+PIGPIO_VERSION = 4862a16c9f76f9b2096055c98ef4fbc480dc1878
+PIGPIO_SITE = $(call github,joan2937,pigpio,$(PIGPIO_VERSION))
+PIGPIO_LICENSE = Unlicense
+PIGPIO_LICENSE_FILES = UNLICENCE
+
+define PIGPIO_BUILD_CMDS
+	$(MAKE) $(TARGET_CONFIGURE_OPTS) VC_DIR=$(STAGING_DIR)/usr -C $(@D) -j4
+endef
+
+define PIGPIO_INSTALL_TARGET_CMDS
```

Comment:
Update the install commands to this:
```
define PIGPIO_INSTALL_STAGING_CMDS
        $(MAKE) $(TARGET_CONFIGURE_OPTS) prefix=/usr DESTDIR=$(STAGING_DIR) -C $(@D) install
endef                     
                          
define PIGPIO_INSTALL_TARGET_CMDS
        $(MAKE) $(TARGET_CONFIGURE_OPTS) prefix=/usr DESTDIR=$(TARGET_DIR) -C $(@D) install
endef                     
```
This will put everything in `/usr` instead of `/usr/local` which will be much more convenient. Buildroot will also automatically clean up the installation of unneeded files now. This will remove the man pages and headers files that currently are in the image.